src/pages/kwonojae/Main/components/Nav.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+import './Nav.scss';
+
+export default class Nav extends Component {
+  render() {
+    const iconList = [
+      { id: 1, className: 'fas fa-home' },
+      { id: 2, className: 'far fa-paper-plane' },
+      { id: 3, className: 'far fa-compass' },
+      { id: 4, className: 'fas fa-heart' },
+    ];
+
+    return (
+      <nav className="navOJ">
+        <div className="navTitle">
+          <span className="main">westargram</span>
+        </div>
+        <div className="navSearch">
+          <input type="text" placeholder="검색" />
+        </div>
+        <div className="navIcons">
+          {iconList.map(iconInfo => {
+            return <i key={iconInfo.id} className={iconInfo.className} />;
+          })}
```

Comment:
```suggestion
          {iconList.map(iconInfo => {
           const {id, className} = iconInfo;
            return <i key={id} className={className} />;
          })}
```
여기도 구조분해 할당 적용해줄 수 있습니다!

Reply:
구조 분해 할당 적용하였습니다~